hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/validators/block/StateChangesValidator.java

```diff
@@ -236,7 +236,8 @@ public StateChangesValidator(
         final var currentVersion = new ServicesSoftwareVersion(servicesVersion, configVersion);
         final var lifecycles = newPlatformInitLifecycle(bootstrapConfig, currentVersion, migrator, servicesRegistry);
         state = new MerkleStateRoot(lifecycles, version -> new ServicesSoftwareVersion(version, configVersion));
-        state.getPlatformState();
+        // initialize the platform state
+        state.getReadablePlatformState();
```

Comment:
⚠️ This needs to be `getWritablePlatformState()` so the validator initializes its platform state in the same relative order as the nodes in the test network.

Reply:
Great! Thank you for the tip, I fixed that.